sources/packages/web/src/components/generic/formio.vue

```diff
@@ -65,8 +65,10 @@ export default defineComponent({
   setup(props, context) {
     const { registerUtilsMethod } = useFormioUtils();
     const { currencyFormatter } = useFormatters();
+    const { mapOfferingIntensity } = useOffering();
     // Register global utils functions.
     registerUtilsMethod("currencyFormatter", currencyFormatter);
+    registerUtilsMethod("mapOfferingIntensity", mapOfferingIntensity);
```

Comment:
No longer required as per [this comment](https://github.com/bcgov/SIMS/pull/4569/files#r2027524245).